### DISCUSSION TITLE
from where and how do i get the satellite images necessary to feed into the model

### DISCUSSION THREAD

**Discussion by s44002** [weecology/DeepForest#658] [Q&A]
from where and how do i get the satellite images necessary to feed into the model

**Comment by macdatar** [comment]
Hello, not that I have an answer but I can ask in the community if you tell me a bit more about what you mean. You want to get some satellite data to then feed it to the release model of deepforest and see the results?

**Reply by s44002** [reply]
I want to use data from satellite to detect tree crowns

**Comment by ethanwhite** [comment]
We haven't really done any satellite work with the package ourselves, though at least [one other group of researchers has](https://www.researchgate.net/profile/Edy-Irwansyah/publication/378008284_Tree_Counting_with_Deep_Forest_Algorithm_for_Kulon_Progo_District_in_Yogyakarta_Indonesia_Using_Pleiades_Satellite_Imagery/links/65c2e0401bed776ae3358928/Tree-Counting-with-Deep-Forest-Algorithm-for-Kulon-Progo-District-in-Yogyakarta-Indonesia-Using-Pleiades-Satellite-Imagery.pdf). They used [data from the Pléiades program](https://earth.esa.int/eogateway/missions/pleiades).

In general there is a chance that these approaches can work with any satellite data of sufficiently high resolution. Much of this data is commercial and so has to be purchased. For example, the very cool recent work by Tucker et al. used Maxar data, but [as they state in the data availability statement it is commercial and can't be shared](https://www.nature.com/articles/s41586-022-05653-6#data-availability). Sometimes this data is available through contracts with funding research agencies, so depending on your situation you may be able to acquire it that way. Maxar is the major commercial provider, but there are others.

The Pléiades data can be [obtained for research use, but requires an application](https://earth.esa.int/eogateway/missions/pleiades#data-section).

Hope this helps!

**Reply by s44002** [reply]
Ethan, the purchase of data is not a problem, at all. we can get that. What resolution?, what should be the technical specifications of the satellite data?.. if you could shed some light on that, i would appreciate it so so much.. we are already partnered with var

**Reply by ethanwhite** [reply]
The short version is that we don't really know because we haven't tried it. The paper I linked above suggests that Pléiades might work (~0.5 m). The Tucker et al. paper suggests that a different set of algorithms works with Maxar data (~0.5 m). That resolution seems reasonable for large trees. You would probably need to fine tune the model with a some hand labeling of the satellite data and figure out the right patch size for the data, but that seems like it would be feasible.

So, we can't say for sure but <=0.5 m resolution seems like it has a decent chance of working (at least for large trees; at least in open canopy areas) with a little bit of fine tuning and hyperparameter selection.
